sdk/servicebus/microsoft-azure-servicebus/pom.xml

```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.microsoft.azure</groupId>
   <artifactId>azure-servicebus</artifactId>
-  <version>3.1.1</version> <!-- {x-version-update;com.microsoft.azure:azure-servicebus;current} -->
+  <version>3.1.3</version> <!-- {x-version-update;com.microsoft.azure:azure-servicebus;current} -->
```

Comment:
You incremented the version by 2. Was it supposed to be `3.1.2`?

Reply:
It was actually 3.1.2. Some branch wide change they made reverted it to 3.1.1.